WalletWasabi.Fluent/Behaviors/SwallowEnterKeyBehavior.cs

```diff
@@ -0,0 +1,21 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Input;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+internal class SwallowEnterKeyBehavior : AttachedToVisualTreeBehavior<InputElement>
+{
+	protected override void OnAttachedToVisualTree(CompositeDisposable disposable)
+	{
+		if (AssociatedObject is null)
+		{
+			return;
+		}
+
+		Observable
+			.FromEventPattern<KeyEventArgs>(AssociatedObject, nameof(InputElement.KeyDown))
```

Comment:
Shouldn't be done the same thing with KeyUp? Just to truly swallow everything.

Reply:
Oh dear... what a mistake! Sorry.
I'm fixing this ASAP...